pkg/option/flags.go

```diff
@@ -205,6 +205,7 @@ func AddFlags(flags *pflag.FlagSet) {
 	flags.String(KeyK8sKubeConfigPath, "", "Absolute path of the kubernetes kubeconfig file")
 	flags.Bool(KeyEnableProcessAncestors, true, "Include ancestors in process exec events")
 	flags.String(KeyMetricsServer, "", "Metrics server address (e.g. ':2112'). Disabled by default")
+	flags.String(KeyMetricsLabelFilter, "", "Metrics Label Filter")
```

Comment:
It's unclear from the help text what the user should pass, could we be more descriptive here? For example:
```
Comma-separated list of enabled metric labels. By default all labels are enabled.
```

Reply:
ok i will change it